lib/commands/performance.js

```diff
@@ -90,6 +95,14 @@ commands.mobileStartPerfRecord = async function (opts = {}) {
     '-D', localPath,
     '-l', `${timeout}`,
   ];
+  if (pid) {
+    if (`${pid}`.toLowerCase() === 'current') {
+      const appInfo = await this.proxyCommand('/wda/activeAppInfo', 'GET');
+      args.push('-p', `${appInfo.pid}`);
```

Comment:
no need for the template string if it's just a variable

Reply:
you mean Subprocess anyway transforms it to a string under the hood?